src/tools/miri/test-cargo-miri/run-test.py

```diff
@@ -8,8 +8,8 @@
 import difflib
 import os
 import re
-import sys
 import subprocess
+import sys
```

Comment:
I think my formatter sorted this by accident... but I guess it's better that way anyway